plugin/trino-redshift/src/test/java/io/trino/plugin/redshift/TestRedshiftCastPushdown.java

```diff
@@ -364,6 +394,16 @@ public void testCastPushdownWithForcedTypedToInteger()
                 .isNotFullyPushedDown(ProjectNode.class);
     }
 
+    @Test
+    void testCastPushdownWithForcedTypedToVarchar()
+    {
+        // These column types are not supported by default by trino. These types are forced mapped to varchar.
+        assertThat(query("SELECT CAST(c_timetz AS VARCHAR(100)) FROM %s".formatted(leftTable())))
+                .isNotFullyPushedDown(ProjectNode.class);
+        assertThat(query("SELECT CAST(c_super AS VARCHAR(100)) FROM %s".formatted(leftTable())))
```

Comment:
A string concat instead of `formatted` method provides syntax highlight. 

> https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#prefer-string-formatting
 Sometimes, if you only need to append something, consider using the + operator.